content/agent/docker/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Basic Docker Agent usage
+kind: documentation
+aliases:
+    - /guides/basic_agent_usage/docker/
+    - /agent/basic_agent_usage/docker/
+---
+
+If you haven't installed the Agent yet, instructions can be found [in the Datadog agent integration page](https://app.datadoghq.com/account/settings#agent/docker). You can also consult our [Official Docker agent 6 image](https://hub.docker.com/r/datadog/agent/).
+
+## Run the Docker agent
+
+Head over to [datadoghq.com](https://app.datadoghq.com/account/settings#agent/docker) to get the official installation guide.
+
+For a simple docker run, get started with:
+
+```
+docker run -d -v /var/run/docker.sock:/var/run/docker.sock:ro \
+              -v /proc/:/host/proc/:ro \
+              -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+              -e DD_API_KEY=<YOUR_API_KEY> \
+              datadog/agent:latest
+```
+
+- [Find here the list of available images for Agent 6](https://hub.docker.com/r/datadog/agent/tags/), Datadog encourages you to always pick the latest version.
+
+The following environment variables are supported:
```

Comment:
I updated https://github.com/DataDog/datadog-agent/blob/master/Dockerfiles/agent/README.md with new envvars and split them in easier to read sections, could you please pick up the diff? 🙇

Reply:
👌